tests/resources/attr/ignores/temporary.conf

```diff
@@ -0,0 +1,15 @@
+[test]
+name=Honor temporary rules
+
+[assert]
+not-ignored=File.txt
+ignored=NewFolder
+ignored=NewFolder/NewFolder
+ignored=NewFolder/NewFolder/File.txt
+
+[gitignore]
+file=/.gitignore
```

Comment:
Does `/` here mean the worktree root? If so why not specify that the path is to be relative to the worktree root and have them bee relative.